flow/connectors/postgres/qvalue_convert.go

```diff
@@ -584,18 +584,22 @@ func customTypeToQKind(typeName string) qvalue.QValueKind {
 // in tstzrange.
 // convertTimeRangeBound removes the +0000 UTC part
 func convertTimeRangeBound(timeBound interface{}) (string, error) {
-	layout := "2006-01-02 15:04:05 -0700 MST"
-	postgresFormat := "2006-01-02 15:04:05"
 	var convertedTime string
-	if timeBound != nil {
-		lowerParsed, err := time.Parse(layout, fmt.Sprint(timeBound))
-		if err != nil {
-			return "", fmt.Errorf("unexpected lower bound value in tstzrange. Error: %v", err)
+	switch timeBound := timeBound.(type) {
+	case pgtype.InfinityModifier:
+		return timeBound.String(), nil
+	default:
+		layout := "2006-01-02 15:04:05 -0700 MST"
+		postgresFormat := "2006-01-02 15:04:05"
+		if timeBound != nil {
+			lowerParsed, err := time.Parse(layout, fmt.Sprint(timeBound))
+			if err != nil {
+				return "", fmt.Errorf("unexpected bound value in tstzrange. Error: %v", err)
+			}
+			convertedTime = lowerParsed.Format(postgresFormat)
+		} else {
+			convertedTime = ""
```

Comment:
```suggestion
			return lowerParsed.Format(postgresFormat), nil
		} else {
			return "", nil
```
no need for `convertedTime`